language/move-core/types/Cargo.toml

```diff
@@ -21,7 +21,8 @@ bcs = { path = "../../../bcs", default-features = false }
 diem-crypto = { path = "../../../crypto/crypto", default-features = false }
 diem-crypto-derive = { path = "../../../crypto/crypto-derive", default-features = false }
 short-hex-str = { path = "../../../common/short-hex-str", default-features = false }
-parity-scale-codec = { version = "1.3.5", features = ["derive"], default-features = false }
+parity-scale-codec = { version = "2", features = ["derive"], default-features = false }
+parity-scale-codec-derive = { version = "2", default-features = false }
```

Comment:
Is it really needed here, as you're using `derive` feature?

Reply:
Oops, should remove derive, as it's nonexistent as feature now.